hooks/post_gen_project.py

```diff
@@ -80,9 +80,16 @@ def create_dotenv() -> None:
         ["cp", ".env.example", ".env"], stdout=sys.stdout, stderr=sys.stderr, check=True
     )
 
+def create_media_dir() -> None:
+    print("Creating media directory")
+    subprocess.run(
+        ["mkdir", "media"], stdout=sys.stdout, stderr=sys.stderr, check=True
+    )
```

Comment:
Can we just add this to the cookiecutter, or does it need to be created after the fact?

Reply:
With the pattern we have now, it'd need to be created after the fact just because the directory itself is being gitignored (since it's intended for local files).

We probably can add it to the cookiecutter, though it might complicate things more. I'd probably add a `.gitkeep` file to the `media` dir so it actually gets kept in the repo and then gitignore all files within that directory _except_ for the `.gitkeep`. Then I'd alter `dump_content` to copy all files except for the `.gitkeep` into the `fixtures` dir - [probably with something like this](https://askubuntu.com/a/333641) - so that doesn't also get included in any fixtures.

I'm leaning towards what we have now to keep it simple, but what do you think?

Reply:
Aha, great observation. I'm fine with your original approach! Thank you. :-)